composer.json

```diff
@@ -11,7 +11,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.2",
-        "symfony/event-dispatcher": "^2.0 || ^3.0"
+        "symfony/event-dispatcher": "^2.3 || ^3.0"
```

Comment:
I understand there is no technical reason. Yet this slows down continuous upgrade process in PHP ecosystem, since it propagates using unsupported version.

Anyone new coming here might see this as 2.3 is fine to use. Which is not. It doesn't even have security fixes anymore.

Also it suggest it's ok to ask same lower version requirement in other packages.

Same goes for PHP version with no security fixes.